packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/Questions/components/RadioInputs/RadioInputs.stories.js

```diff
@@ -39,12 +39,13 @@ export default {
   component: RadioInputs
 }
 
-const Template = ({ dark, options, questionId }) => (
+const Template = ({ dark, options, questionAnswer, questionId }) => (
   <StoryContext
     theme={{ ...zooTheme, dark }}
   >
     <RadioInputs
       options={options}
+      questionAnswer={questionAnswer}
```

Comment:
This doesn't demonstrate the new behavior here, because the story doesn't use a store? Maybe a future update can be to include the store in the story context? Not blocking though.

Reply:
That's correct, and an update with a store would be very helpful. Similarly, I was thinking maybe a full survey task (with store) might be helpful.